build/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM quay.io/kubernetes-ingress-controller/debian-base-amd64:0.1
+FROM gcr.io/google-containers/debian-base-amd64:0.4.0
```

Comment:
Please don't change this. The image 0.1 uses debian buster while 0.4.0 debian strech

Reply:
The old image was failing for me with can't find libipset3.

Reply:
Ok, I reverted to original base image and fixed docker build by switching libipset3 with libipset11.